scripts/superdesk-authoring/views/change-image.html

```diff
@@ -16,6 +16,10 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
                 <input type="text" ng-model="data.metadata.description_text" ng-change="data.isDirty = true" class="line-input">
             </div>
             <div>
+                <label translate>Alt Text</label>
```

Comment:
Exactly at the same place where you set it.
On master it should appear only for featured images in a story. So you can edit it by clicking on the image.